src/future/formatic-fields/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
Moved this all out of `/index.js` to group all of our form field components.

Reply:
Works for me. Curious: why `formatic-fields` and not just `fields`?

Reply:
My thought was we'll probably have a `Fields` component here, and it might be confusing to have `/fields/Fields.js`.

Reply:
Ah, gotcha. That works then. Maybe something like `builtin-fields` would work too? Just thinking out loud, `formatic-fields` is fine, just wondering if others might wonder what it means too.

Reply:
I'm good with `builtin-fields`. Just didn't want it to be simply `fields`.